compliance/disallowed_images/disallowed_cloud_images.pt

```diff
@@ -0,0 +1,260 @@
+    if ( param_exclude_tags.length > 0 ){
+      var filtered_tags = "&tag!=" + param_exclude_tags.join(",");
+    }
+	request = {
+      "auth": "auth_rs",
+      "verb": "GET",
+      "host": rs_cm_host,
+      "path": "/api/instances",
+      "headers": {
+        "X-Api-Version": "1.6",
+        "X-Account": \""+rs_project_id+\"",
+        "accept": "application/json"
+      },
+      "query_params": {
+        "view": "full",
+        "filter": "state=running,operational,provisioned" + filtered_tags
+      }
+    }
+  EOS
+end
+
+script "js_image_list", type: "javascript" do
+  parameters "rs_cm_host","cloud_id","rs_project_id"
+  result "request"
+  code <<-EOS
+    request = {
+      "auth": "auth_rs",
+      "verb": "GET",
+      "host": rs_cm_host,
+      "path": "/api/clouds/"+cloud_id+"/images",
+      "headers": {
+        "X-Api-Version": "1.6",
+        "X-Account": \""+rs_project_id+\"",
+        "accept": "application/json"
+      }
+    }
+  EOS
+end
+
+script "js_filter_instances_cloud_id", type: "javascript" do
+ parameters "ds_instances_list"
+ result "content"
+ code <<-EOS
+ var content=[]
+   _.each(ds_instances_list, function(instance){
+      if(instance["image_href"] != null){
+        content.push(instance.cloud_id)
+      }
+    })
+        content = _.uniq(content);
+ EOS
+end
+
+script "js_filter_instances_list", type: "javascript" do
+  parameters "ds_instances_list","ds_images_list","param_allowed_cloud_images"
+  result "content"
+  code <<-EOS
+  var content=[]
+    for ( i =0; i < ds_instances_list.length; i++ ){
+      var instance=ds_instances_list[i]
+        if(instance.image_href != null){
+        // Constructing tags with comma separated to display in detail_template
+         var tags = instance.tags
+         var dis_tags=""
+    for(var j=0;j<tags.length;j++){
+         var tag=tags[j];
+         dis_tags=dis_tags+", "+tag;
+    }         
+         var instance_image_href = instance.image_href
+	     var matched_image = _.find(ds_images_list, function(image) {return image.href == instance_image_href})
+	    if (matched_image != null && matched_image !== undefined) {
+	     var resource_uid = matched_image.resource_uid
+         var is_cloud_image_resource_uid_matched=false
+        if((param_allowed_cloud_images.indexOf(resource_uid) !== -1) || (param_allowed_cloud_images.join()).match(resource_uid)){
+         is_cloud_image_resource_uid_matched = true;
+        }
+        }
+        if(matched_image != null && !(is_cloud_image_resource_uid_matched)){
+            content.push({
+             cloud_name:instance['cloud_name']
+             id:instance['id']
+             cloud_type:instance['cloud_type']
+             name:instance['name']
+             tags:(dis_tags.slice(2))
+             href:instance['href']
+             locked:instance['locked']
+             state:instance['state']
+            })   
+	    }
+	    }
+	}
+	    content.sort(getSortOrder("cloud_name"));
```

Comment:
You should use the underscorejs sortBy function instead of using your own.  
https://underscorejs.org/#sortBy
https://github.com/flexera/policy_templates/blob/master/saas/fsm/renewal_reminder/fsm-renewal_reminder.pt#L178

Reply:
Now I have changed function to underscorejs